### PR TITLE
Fix Banner styling & close button

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

This PR:

- Gets the changes from #186  into storybook
- Includes the icon styling fix from #189.
- Fixes the bug where the close button in the `<Banner>` wasn't working ([support thread](https://dsva.slack.com/archives/CBU0KDSB1/p1633020309310600))

## Testing done

Storybook :eyes: 

## Screenshots

<details>
<summary>Banner closing</summary>

Peek 2021-09-30 10-46
</details>

<details>
<summary>Full width alert closing</summary>

Peek 2021-09-30 10-49
</details>

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -99,7 +99,7 @@
     "sass-loader": "^8.0.2",
     "sinon": "^9.2.2",
     "uswds": "1.6.10",
-    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-v0.10.0"
+    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-fullwidth-close"
```
Comment (bkjohnson) [inline package.json:102]: We can update this once we have a release for #189

File: src/components/Banner/Banner.jsx [hunk 2]
```diff
@@ -136,7 +136,7 @@ export class Banner extends Component {
           visible
           full-width
           closeable={showClose}
-          onCloseAlert={onCloseAlert}
+          onClose={onCloseAlert}
```
Comment (bkjohnson) [inline src/components/Banner/Banner.jsx:139]: `close` is the event name, not `closeAlert`.

[Slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1633020309310600).

File: src/components/Banner/Banner.stories.jsx [hunk 3]
```diff
@@ -19,6 +19,7 @@ const Template = args => <Banner {...args} />;
 const defaultArgs = {
   visible: true,
   title: 'Coronavirus',
+  storage: { getItem: () => {}, setItem: () => {} },
```
Comment (bkjohnson) [inline src/components/Banner/Banner.stories.jsx:22]: The Banner is designed not to close without this prop :shrug: 

https://github.com/department-of-veterans-affairs/component-library/blob/07dd499dbdb5c6b5f13b03977167f019adbd2bf6/src/components/Banner/Banner.jsx#L130-L131

**Review by bkjohnson — COMMENTED** [review]
This includes some more changes related to alerts and banners.

**Review by cvalarida — APPROVED** [review]

**Pull request merged**